docs/scss/_community.scss

```diff
@@ -0,0 +1,25 @@
+.ov-community {
+    display:flex;
+    flex-direction: column;
+    > *:not(:last-child) {
+    margin-bottom: var(--ov--spacer--md);
+    }
+    }
+
+.ov-masthead--community {
+	    background: url('/images/oVirt_Community_Header.png') left top no-repeat var(--ov--color-brand--100);
+    .ov-heading--lg {
+        max-width: 400px;
+        margin-left: auto;
+        margin-right: auto;
+    } 
+   }
+
+
+.ov-community-paragraph {
+	max-width: 1080px; 
+    }    
+.ov-community-list {
```

Comment:
Same for this, we can talk more about this. I don't think you'll need a width here and if you are trying to indent uls do that through the content styles wrapper.  Also, I'd recommend using the spacer system for the margin, if there isn't one there you need you can add it.

Reply:
Yeah I was trying to indent the uls here. I'll try to play around with content styles wrapper since I don't think I've tried that yet. 

For using the spacer system for the margins would the attribute with the value look something like: `margin: --ov--spacer--sm;`?